taca/analysis/analysis.py

```diff
@@ -1,246 +1,218 @@
+        logger.warn("Cannot find the runParameters.xml file at {}. " + 
+                    "This is quite unexpected. please archive the run {} manually".format(rppath, run))
     else:
-        #this information about the run type (with HiSeq2.5 applicationaName does not work anymore, but as for a long time we will have instruments not updated I need to find out somehting that works
+        # This information about the run type (with HiSeq2.5 applicationaName does not work anymore, 
+        # but as for a long time we will have instruments not updated I need to find out something that works
         try:
-            #Works for recent control software
-            runtype=rp.data['RunParameters']["Setup"]["Flowcell"]
+            # Works for recent control software
+            runtype = rp.data['RunParameters']["Setup"]["Flowcell"]
         except KeyError:
-            #use this as second resource but print a warning in the logs
-            logger.warn("Parsing runParameters to fecth instrument type, not found Flowcell information in it. Using ApplicaiotnName")
-            runtype=rp.data['RunParameters']["Setup"].get("ApplicationName", "") # here makes sense to use get with default value "" -> so that it doesnt raise an exception in the next lines (in case ApplicationName is not found, get returns None)
-
+            # Use this as second resource but print a warning in the logs
+            logger.warn("Parsing runParameters to fecth instrument type, " + 
+                        "not found Flowcell information in it. Using ApplicaiotnName")
+            # here makes sense to use get with default value "" -> 
+            # so that it doesn't raise an exception in the next lines 
+            # (in case ApplicationName is not found, get returns None)
+            runtype=rp.data['RunParameters']["Setup"].get("ApplicationName", "") 
+
+        # Jose : return the run object here instead
         if "HiSeq X" in runtype:
-            return 'HiSeqX'
+            return HiSeqX_Run(run, CONFIG["analysis"]["HiSeqX"])
         elif "MiSeq" in runtype:
-            return 'MiSeq'
+            return HiSeq_Run(run, CONFIG["analysis"]["HiSeq"])
         elif "HiSeq" in runtype or "TruSeq" in runtype:
-            return 'HiSeq'
+            return MiSeq_Run(run, CONFIG["analysis"]["MiSeq"])
+        elif "NextSeq" in runtype:
+            return NextSeq_Run(run, CONFIG["analysis"]["NextSeq"])
         else:
-            logger.warn("unrecognized runtype {}, cannot archive the run {}. Someone as likely bought a new sequencer without telling it to the bioinfo team".format(runtype, run))
-            return None
+            logger.warn("Unrecognized run type {}, cannot archive the run {}. " + 
+                        "Someone as likely bought a new sequencer without telling " + 
+                        "it to the bioinfo team".format(runtype, run))
```

Comment:
I'm fine with people following pep8, but not with breaking up a string  and using concatenation. Either use the backslash or the implicit concatenation.